image_width: 2048
image_height: 1536
camera_name: bfly
camera_matrix:
  rows: 3
  cols: 3
  data: [1826.42854934, 0.0, 1002.25449315, 0.0, 1825.80721561, 787.476743745, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.182942903269, 0.107077802795, 1.15955071853e-05, 0.000119029626267, 0.0]
